<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IEC104 Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/new/prefix1/icons/Q104.png</normalon>
   </iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit_nameFilter"/>
    </item>
    <item>
     <widget class="QTableView" name="MTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="MTool_add">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add new tag</string>
        </property>
        <property name="text">
         <string notr="true">+</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/prefix1/icons/plus.png</normaloff>:/new/prefix1/icons/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="MTool_remove">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove tag</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/prefix1/icons/minus.png</normaloff>:/new/prefix1/icons/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_SaveBase"/>
    <addaction name="action_LoadFile"/>
    <addaction name="action_LoadBase"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Command</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSettings"/>
   <addaction name="Interrogation"/>
   <addaction name="actionTS"/>
   <addaction name="actionCMD"/>
   <addaction name="actionWatch"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="clearLog">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Clear log</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/icon_erase</normalon>
          </iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="log">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icon_connect</normaloff>:/new/prefix1/icon_connect</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icon_disconnect</normaloff>:/new/prefix1/icon_disconnect</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icon_settings</normaloff>:/new/prefix1/icon_settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_SaveBase">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="action_LoadFile">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_params">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="Interrogation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/icons/interrogation.png</normaloff>:/new/prefix1/icons/interrogation.png</iconset>
   </property>
   <property name="text">
    <string>IC</string>
   </property>
   <property name="toolTip">
    <string>General interrogation</string>
   </property>
  </action>
  <action name="actionTS">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/icon_timer</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>TS</string>
   </property>
   <property name="toolTip">
    <string>Time sync</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionCMD">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/icons/cmd.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>CMD</string>
   </property>
   <property name="toolTip">
    <string>Commands</string>
   </property>
  </action>
  <action name="action_LoadBase">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWatch">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/icons/watch.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Таблица просмотра</string>
   </property>
   <property name="toolTip">
    <string>Watch table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnConnectPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnSettingsPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisconnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnDisconnectPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearLog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnClearLogPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnSettingsPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Interrogation</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnGIPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MTool_remove</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>MToolRemove_Pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MTool_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>MToolAdd_Pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnConnectPressed()</slot>
  <slot>OnDisconnectPressed()</slot>
  <slot>OnSettingsPressed()</slot>
  <slot>OnClearLogPressed()</slot>
  <slot>OnGIPressed()</slot>
  <slot>MToolRemove_Pressed()</slot>
  <slot>MToolAdd_Pressed()</slot>
 </slots>
</ui>
